Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -2226,14 +2228,15 @@ class CVE(Indicator):
         """
         CONTEXT_PATH = 'CVE(val.ID && val.ID == obj.ID)'
 
-        def __init__(self, id, cvss, published, modified, description):
+        def __init__(self, id, cvss, published, modified, description, dbot_score):
             # type (str, str, str, str, str) -> None
 
             self.id = id
             self.cvss = cvss
             self.published = published
             self.modified = modified
             self.description = description
+            self.dbot_score = dbot_score
```

Comment:
```
self.dbot_score = Common.DBotScore(
    indicator=id,
    indicator_type=DBotScoreType.CVE,
    integration_name=integration_name,
    score=Common.DBotScore.NONE
)

```
add new helper function in commonserverpython
``` 
def get_integration_name():
    return demisto.callingContext.get('IntegrationBrand')
```

In DBotScore class constructor 
```
self.integration_name = integration_name or get_integration_name()
```